package/gluon-web-admin/luasrc/lib/gluon/web/model/admin/remote.lua

```diff
@@ -30,80 +31,87 @@ function keys:write(value)
 	end
 end
 
+if ((site.config_mode or {}).remote_login or {}).allow_password_login ~= false then
```

Comment:
I thought about this earlyer, but then I thought it wouldn't be good readeable to exit the script so early with return. But fine ;=

done